expression/builtin_time.go

```diff
@@ -6446,10 +6450,7 @@ func (b *builtinTimeFormatSig) evalString(row chunk.Row) (string, bool, error) {
 
 // formatTime see https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_time-format
 func (b *builtinTimeFormatSig) formatTime(ctx sessionctx.Context, t types.Duration, formatMask string) (res string, err error) {
-	t2 := types.NewTime(types.FromDate(0, 0, 0, t.Hour(), t.Minute(), t.Second(), t.MicroSecond()), mysql.TypeDate, 0)
-
-	str, err := t2.DateFormat(formatMask)
-	return str, err
+	return t.DateFormat(formatMask)
```

Comment:
Why not convert `Duration` to `Time` and use `Time.DateFormat`?  : )

Reply:
Because Time only have 5 bit for hour, but hour in Duration can reach 838.

Reply:
OK.